qos/evm/service_state.go

```diff
@@ -162,3 +163,53 @@ func (ss *serviceState) updateFromEndpoints(updatedEndpoints map[protocol.Endpoi
 
 	return nil
 }
+
+func (ss *serviceState) hydrateDisqualifiedEndpointsResponse(serviceID protocol.ServiceID) devtools.QoSLevelDataResponse {
+	ss.logger.Info().Msgf("Hydrating disqualified endpoints response for service ID: %s", serviceID)
+
+	disqualifiedEndpoints := make(map[protocol.EndpointAddr]devtools.QoSDisqualifiedEndpoint)
+	emptyResponseCount := 0
```

Comment:
Can you #PUC what the plan is here for other checks we'll add?

- DO we need to update this?
- Is this all we'll ever have?
- Is there an enum?